lib/elixir_boilerplate_web/home/templates/index.html.eex

```diff
@@ -3,6 +3,8 @@
   </a>
 
-  <p>This repository is the stable base upon which we build our Elixir projects at Mirego.<br />We want to share it with the world so you can build awesome Elixir applications too.</p>
-  <p><strong>Message:</strong> <%= @message %></p>
+  <p <%= testid_attribute("description-paragraph") %>>This repository is the stable base upon which we build our Elixir projects at Mirego.<br />We want to share it with the world so you can build awesome Elixir applications too.</p>
+  <%= content_tag :p, data_testid: testid_key("message-paragraph") do %>
```

Comment:
Doesn't this defy the purpose since `data-testid=""` will end up in the DOM in production?

Reply:
if an attribute has `nil` value it will not be added in resulting `p` html tag